client/src/DbtLanguageClientManager.ts

```diff
@@ -108,20 +121,21 @@ export class DbtLanguageClientManager {
         this.manifestParsedEventEmitter,
         this.statusHandler,
       );
+      this.clients.set(projectUri.path, client);
```

Comment:
Nice catch. If we can add unit test on it?

Reply:
I think we can. It will require some time to prepare mocha for client, I will do it later.